ui/src/message_popup/experience_page.jsx

```diff
@@ -128,17 +128,7 @@ export default React.createClass({
     
     return (
       <div>
-        <AppBar 
-          title='Message PopUp Practice'
-          iconElementLeft=
-          {
-            <IconButton
-              onTouchTap={this.resetExperience}
-              >
-              {!hasStarted ? <MenuIcon/> : questionsAnswered >= sessionLength ? <RefreshIcon /> : <ArrowBackIcon />}
-            </IconButton>
-          }
-          />
```

Comment:
@david-rosales This didn't translate as well with the navigation menu on the left (e.g, having the back button on the right side menu).  I removed it to simplify for now, but if you have any ideas to improve on this that'd be great too!